Modules/SQLDatabase/2.0/Tests/parameters.json

```diff
@@ -7,6 +7,12 @@
         },
         "serverName": {
             "value": "bestesqlserver0"
+        },
+        "diagnosticStorageAccountId": {
+            "value": "foo"
```

Comment:
Same as above.  Should this be  based on standard naming convention?

Reply:
Dummy values.